projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/jupyter_sample_job/10_notebook.ipynb

```diff
@@ -1,35 +1,90 @@
 {
- "cells": [],
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16a6ea87-7142-44dd-94b6-e95fe2d57d06",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "vdk_ipython extension introduces a magic command for Jupyter.\n",
+    "The command enables the user to load VDK for the current notebook.\n",
+    "VDK provides the job_input API, which has methods for:\n",
+    "    * executing queries to an OLAP database;\n",
+    "    * ingesting data into a database;\n",
+    "    * processing data into a database.\n",
+    "See the IJobInput documentation for more details.\n",
```

Comment:
Can you please link to the documentation?

Reply:
@gageorgiev https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/src/vdk/api/job_input.py
we do not have other documentation so link this i have done so for the transform job